src/main/java/demo/userboard/service/UserServiceImpl.java

```diff
@@ -33,6 +32,18 @@ public JoinResponseDto join(JoinRequestDto requestDto) {
         return new JoinResponseDto("성공", savedUser.getId(), 200);
     }
 
+    @Override
+    public LoginResponseDto login(LoginRequestDto requestDto) {
+        Optional<User> user = userRepository.findByEmail(requestDto.getEmail());
+        if (user.isPresent()) {
+            User user1 = user.get();
+            if (user1.getPassword().equals(requestDto.getPassword())) {
+                return LoginResponseDto.from(user1);
+            }
+        }
+        return null;
+    }
```

Comment:
exception 처리를 Service단으로 내리면 이렇게 가능하겠네요

```suggestion
    @Override
    public LoginResponseDto login(LoginRequestDto requestDto) {

        return userRepository.findByEmail(requestDto.getEmail())
            .filter(user -> user.getPassword().equals(requestDto.getPassword()))
            .map(LoginResponseDto::from)
            .orElseThrow(() -> new CustomException(CustomErrorCode.UNAUTHORIZED));
    }
```

Reply:
@evan-song2 로그인 메서드 수정 완료했습니다.[cd27588](https://github.com/PNoahKR/user-board/pull/8/commits/cd275883397a44ab9a60513100c733490391947a)